readme.md

```diff
@@ -56,107 +56,117 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** What are headers? What values do you recognize?
 
-* **Your Answer:**
+* **Your Answer:** Expresss is powering it, the `Content-Type` looks familiar, and the `Date` information.
 
 ---
 
 - [ ] Add the following line **above** the `res.json()` line in your `app.js` file; then, make the request again.
   ```js
-  console.log(req.headers)
+  console.log(req.headers);
   ```
 
 * **Question:** Where can you find this `console.log()` statement? How can you change the headers that are sent in Postman?
 
-* **Your Answer:**
+* **Your Answer:** You will find it in your terminal.
 
 ---
 
 - [ ] Add the following line **above** the `res.json()` line in your `app.js` file; then, make a request again but this time change the URL to the following: `http://localhost:5000?course=javascript`
   ```js
-  console.log(req.query)
+  console.log(req.query);
   ```
 
 * **Question:** What are query parameters? Try going to a couple of your favorite websites and paste an example of query parameters being used.
 
-* **Your Answer:**
+* **Your Answer:** It's a way to define additional information when you make a `GET` request. They are often used to define or specify information in a request.
 
 ---
 
 - [ ] Before all of your routes, add the following:
   ```js
   app.use((req, res, next) => {
-    console.log('In the server!')
-    next()
-  })
+    console.log("In the server!");
+    next();
+  });
   ```
 
 * **Question:** When does `app.use()` get called?
 
-* **Your Answer:**
+* **Your Answer:** `app.use()` is a general function for every request to go through and is called when it finds a match for a specified path.
```

Comment:
Remember app.use will always be called if a request has reached that point of the route chain.